.github/workflows/release.yaml

```diff
@@ -0,0 +1,292 @@
+name: Release build
+
+on:
+  push:
+    tags:
+      - 1*
```

Comment:
Does this mean "any tag starting with 1"? If so, we probably want to be more specific with something like a regex.

Reply:
Updated